dastardcommander/trigger_config.py

```diff
@@ -443,6 +448,20 @@ def changedAutoTrigConfig(self):
             newstate["AutoDelay"] = nsdelay
         except ValueError:
             pass
+
+        # Check the auto veto level spin box. Turn text black if 0, red otherwise.
+        avr = self.autoVetoRange.value()
+        newstate["autoVetoRange"] = avr
```

Comment:
The line above is what makes sure that the `self.autoVetoRange.value()` finds its way to Dastard.